app/src/main/java/com/pvp/app/ui/screen/settings/Settings.kt

```diff
@@ -14,15 +14,22 @@ import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.outlined.Badge
 import androidx.compose.material.icons.outlined.Notifications
 import androidx.compose.material3.Card
-import androidx.compose.material3.Divider
+import androidx.compose.material3.HorizontalDivider
 import androidx.compose.material3.Icon
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.vector.ImageVector
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.setValue
+import com.pvp.app.ui.common.Picker
+import com.pvp.app.ui.common.PickerState.Companion.rememberPickerState
+
+var reminderTimeMinutes by mutableIntStateOf(10)
```

Comment:
This is implemented really weird, can it be just an int of `10`, because in the end you are just getting the `int`, right?

It also should only be used, if user has not set their value for tasks reminder, else we would take the actual value user has set from before.

Upon setting change the value should be persisted into the database.